lambdas/log-redaction/src/redact-handler.ts

```diff
@@ -79,6 +79,14 @@ export class RedactHandler implements LambdaInterface {
       throw error;
     }
   }
+
+  private formatMessage(message: string) {
+    try {
+      return JSON.stringify(JSON.parse(message), null, 2);
+    } catch (error) {
+      return message;
```

Comment:
What is the scenario where this isn't JSON? Could it contain something sensitive?

Reply:
Oh wait ignore me